prow/plugins/cat/cat_test.go

```diff
@@ -40,6 +40,8 @@ var category = flag.String("category", "", "Request a particular category if set
 var movieCat = flag.Bool("gif", false, "Specifically request a GIF image if set")
 var keyPath = flag.String("key-path", "", "Path to api key if set")
 
+const grumpyImage = "https://upload.wikimedia.org/wikipedia/commons/e/ee/Grumpy_Cat_by_Gage_Skidmore.jpg"
```

Comment:
You don't need to redefine this. You can just use the `grumpyURL` variable that is defined in `cat.go` because it is in the same package (`cat`).